cmake/ruy_include_directories.cmake

```diff
@@ -14,20 +14,8 @@
 
 function(ruy_include_directories NAME DEPS)
   target_include_directories(${NAME}
-      PUBLIC
-      "${PROJECT_SOURCE_DIR}"
+    PUBLIC
+      "$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}>"
+      "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>"
   )
-  if (cpuinfo IN_LIST DEPS)
-    target_include_directories(${NAME}
-      PRIVATE
-        "${PROJECT_SOURCE_DIR}/third_party/cpuinfo/include"
-    )
-  endif()
-  if ((gtest IN_LIST DEPS) OR
-      (gtest_main IN_LIST DEPS))
-    target_include_directories(${NAME}
-      PRIVATE
-        "${PROJECT_SOURCE_DIR}/third_party/googletest/googletest"
-    )
-  endif()
```

Comment:
Why is this not needed anymore?

Reply:
I'm not sure why this was needed in the first place since both `cpuinfo` and `gtest` targets set the include directory property which applies to their dependents so this should be unnecessary.